src/sonic-yang-mgmt/yang-models/sonic-head.yang

```diff
@@ -0,0 +1,72 @@
+module sonic-head {
+
+    namespace "http://sonic-head";
+    prefix sonic-head;
+
+    organization "Linkedin Corporation";
+
+    contact "lnos_coders@linkedin.com";
+
+    description "Head yang Module for SONiC OS";
+    
+    revision 2019-07-01 {
+        description "First Revision";
+    }
+    
+    typedef ip-family {
+        type enumeration {
+            enum IPv4;
+            enum IPv6;
+        }
+    }
+
```

Comment:
These definitions are already committed by Broadcom.
For e.g. admin-status defined in sonic-mgmt-framework/models/yang/sonic/common/sonic-common.yang

Here it is admin_status and there it is admin-status.

Can we sync ?

Reply:
I will keep same name as Config DB in my PR.